gh-deployments-environments/helper.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+
+set -e
+
+CMD=${1:-};
+
+
+print_syntax() {
+  echo "helper.sh <command> [args...]"
```

Comment:
Suggestion: use below syntax
```
function usage(){
  cat << EOF
Usage:
  $0 <command> [args...]
   ....
EOF
  exit 1
}
```

Reply:
Ah silly me!  Fixed.  I'll resolve this conversation, it's approved and I'd like to use that tiny github action already in one tiny workflow we have that tidies up featurenets.